docs/Cleanup.md

```diff
@@ -0,0 +1,22 @@
+# Cleanup All Resource
+
+To cleanup a Moodle deployment simply delete the Resource Group that
+contains it. The commands below will iterate over your workspace
+directory and delete all deployments.
+
+## Prerequisites
+
+First we need to ensure our [environment variables](./Environment-Variables.md) are correctly configured.
+
+## Remove each resource group
+
+This command will delete all resources in *all* resource groups. Run
+with caution.
+
+Note, that this command will not fully delete the resource group if
+you have Azure Backup enabled since the Recovery Services Vault will
+not be deleted (it's got the backups of you data!).
+
+``` bash
+for filename in $MOODLE_AZURE_WORKSPACE/*; do az group delete --yes --name $(basename $filename) & done
```

Comment:
'& done' at the end of the line? '; done'?

Reply:
The '&' spawns separate processes for each of the delete operations because they can be very long running. When used in a single line statement like this it replaces the ';'

Reply:
I thought we still need ';' even in that case, no? Well, I always felt shell grammar can be so arbitrary anyway... :)

Still, I think the better way is to pass the `--no-wait` flag to the `az group delete` command. That'll make the command unblocking (async, returning immediately after REST request is sent), achieving what you intended, I suppose.

Reply:
Yes, it's a bash shell grammar thing. If you put the ';' in it breaks.

I didn't know about --no-wait, I'll change to that - thanks.